### ISSUE TITLE
Error updating www/config.xml: Error:  Your config.xml file does not have a <content> element

### ISSUE THREAD

**Issue by yreme** [driftyco/ng-cordova#894]
every time  I run    `demo` with command   
```shell
$ ionic serve
```
it will report 
```shell
Error updating www/config.xml: Error: 
Your config.xml file does not have a <content> element. 
Add something like: <content src="index.html"/> (CLI v1.6.1)
```
yet  ` <content src="index.html" />  `  is  already in  `www/config.xml`

By the way,  `phonegap  serve` would not report this .

How could I solve this issue? Is there another way that I could run this  demo?

**Comment by gortok** [comment]
I'll take a look, thanks for reporting this.

**Comment by mhartington** [comment]
Can you post your config.xml?

**Comment by yreme** [comment]
Thanks for reply! The `demo`  I mentioned is in `https://github.com/driftyco/ng-cordova/tree/master/demo`,  while   `https://github.com/driftyco/ng-cordova/blob/master/demo/config.xml` is  `config.xml`

Here  is the error message:
```shell
$ionic serve



Error updating www/config.xml: Error: 
Your config.xml file does not have a <content> element. 
Add something like: <content src="index.html"/> (CLI v1.6.1)

Your system information:

Cordova CLI: 5.1.1
Gulp version:  CLI version 3.9.0
Gulp local:   Local version 3.4.0
Ionic Version: 1.0.0-beta.14
Ionic CLI Version: 1.6.1
Ionic App Lib Version: 0.3.3
OS: Distributor ID:	Ubuntu Description:	Ubuntu 14.04.2 LTS 
Node Version: v0.12.7
```

**Comment by yreme** [comment]
when I  add `<content src="index.html" />` in `https://github.com/driftyco/ng-cordova/blob/master/demo/www/config.xml`  as 
```
<?xml version="1.0" encoding="UTF-8"?>
	<widget xmlns = "http://www.w3.org/ns/widgets"
		xmlns:gap = "http://phonegap.com/ns/1.0"
		id        = "com.ionicframework.ngCordovaDemoUnique"
		versionCode=""
		version   = "">
		<name>ngCordovaDemo</name>
		<description>Demo app for ngCordova plugins		</description>
		<content src="index.html" />
	</widget>
```
then  run  `ionic serve`  , it works \(^o^)/~!!

**Comment by pavan777** [comment]
I uploaded the files to phonegap build, and ran the app on android mobile. It was showing fine but all mobile features like camera, battery etc are not working

**Comment by yreme** [comment]
yes `phonegap serve`  works， but ` ionic serve` would report above error!

**Comment by Joeeeeh7** [comment]
What yreme says, but a lot of plugins doesn't seem to work with me? In log i get errors like: 

[phonegap] [console.error] Error: undefined is not an object (evaluating '$window.datePicker.show')
[phonegap] show@file:///var/mobile/Containers/Data/Application/09550BAE-D120-4953-A059-11A169F1FE3F/Library/files/phoneg
apdevapp/www/lib/ngCordova/dist/ng-cordova.js:1527:27
[phonegap] pick@file:///var/mobile/Containers/Data/Application/09550BAE-D120-4953-A059-11A169F1FE3F/Library/files/phoneg
apdevapp/www/app/datePicker/datePicker.ctrl.js:24:30
[phonegap] $parseFunctionCall@file:///var/mobile/Containers/Data/Application/09550BAE-D120-4953-A059-11A169F1FE3F/Librar
y/files/phonegapdevapp/www/lib/ionic/js/ionic.bundle.js:20124:23
[phonegap] file:///var/mobile/Containers/Data/Application/09550BAE-D120-4953-A059-11A169F1FE3F/Library/files/phonegapdev
app/www/lib/ionic/js/ionic.bundle.js:50863:21
[phonegap] $eval@file:///var/mobile/Containers/Data/Application/09550BAE-D120-4953-A059-11A169F1FE3F/Library/files/phone
gapdevapp/www/lib/ionic/js/ionic.bundle.js:22178:28
[phonegap] $apply@file:///var/mobile/Containers/Data/Application/09550BAE-D120-4953-A059-11A169F1FE3F/Library/files/phon
egapdevapp/www/lib/ionic/js/ionic.bundle.js:22276:28
[phonegap] file:///var/mobile/Containers/Data/Application/09550BAE-D120-4953-A059-11A169F1FE3F/Library/files/phonegapdev
app/www/lib/ionic/js/ionic.bundle.js:50862:19
[phonegap] eventHandler@file:///var/mobile/Containers/Data/Application/09550BAE-D120-4953-A059-11A169F1FE3F/Library/file
s/phonegapdevapp/www/lib/ionic/js/ionic.bundle.js:10823:25
[phonegap] dispatchEvent@[native code]
[phonegap] triggerMouseEvent@file:///var/mobile/Containers/Data/Application/09550BAE-D120-4953-A059-11A169F1FE3F/Library
/files/phonegapdevapp/www/lib/ionic/js/ionic.bundle.js:2811:20
[phonegap] tapClick@file:///var/mobile/Containers/Data/Application/09550BAE-D120-4953-A059-11A169F1FE3F/Library/files/ph
onegapdevapp/www/lib/ionic/js/ionic.bundle.js:2800:20
[phonegap] tapTouchEnd@file:///var/mobile/Containers/Data/Application/09550BAE-D120-4953-A059-11A169F1FE3F/Library/files
/phonegapdevapp/www/lib/ionic/js/ionic.bundle.js:2918:13

Is this problem only appearing to me or..?

**Comment by gortok** [comment]
Hm.  I'll dig into this deeper; I'm able to reproduce this issue.

**Comment by Joeeeeh7** [comment]
This is what i get when i use phonegap serve, when i'm building it by build.phonegap.com and run it on an android device, theres no plugin which does work. Seems like something is wrong!

**Comment by masterom77** [comment]
i am also getting the same error i,e  "the config.xml file is missing a content element that defines the start page" , when run on Win 10 PC , and i have VS 2017 with cordova. Please  sort this issue as soon as possible.
